docs/nodes/GENERATORS/SIMULATIONS/SECOND_ORDER_SYSTEM/SECOND_ORDER_SYSTEM.md

```diff
@@ -1,3 +1,10 @@
+---
+title: SECOND_ORDER_SYSTEM
+description: The SECOND_ORDER_SYSTEM has a second order exponential function. This node is designed to be used in a Python loop . The data is appended as the loop progress and written to memory.
```

Comment:
Uneeded space before the period after the second sentence `'This node is designed to be used in a Python loop'`